sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -183,11 +210,31 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
         springMethods.exists(s => p.serviceMethod.methodName.equals(s.serviceMethod.methodName)))
     }
 
+    val serviceLevelOptions = {
+
+      val allOptions =
+        AclDescriptorFactory.serviceLevelAclAnnotation(component) ::
+        eventingInForEventSourcedEntityServiceLevel(component) ::
```

Comment:
If I understood correctly now when annotating the class this has to be signaled in the gRPC annotation at the service level. Is this the case for all annotations? Because I'm missing here subscription to a topic.

Reply:
I didn't touch the topic part because not relevant for this feature. 

We should indeed ensure that a type level subscription to a topic becomes a gRPC service level annotation, but I prefer to do it later since not relevant for now.